### ISSUE TITLE
Invalid option 't' while resuming transfer

### ISSUE THREAD

**Issue by adrianmarcu18** [yboetz/pyznap#71]
Hello. Trying to use the new functionalities of retry and resume, but having some issues while resuming. The output looks something like this (replaced the actual destination and dataset name):

```
Sep 25 09:33:34 INFO: Found resume token. Resuming last transfer of sshdestination:dataset (~0.0B)...
  invalid option 't'
usage:
        send [-DnPpRvLe] [-[iI] snapshot] <snapshot>
        send [-Le] [-i snapshot|bookmark] <filesystem|volume|snapshot>

For the property list, run: zfs set|get

For the delegated permission list, run: zfs allow|unallow
Sep 25 09:33:34 ERROR: Error while sending to sshdestination:dataset: cannot receive: failed to read from stream...
```

Later edit: Trying to remove the resume option from the config and run again the transfer will fail with the same error. So even though the resume config option is removed, it will still try to resume the last failed transfer and I am stuck in a loop.

**Comment by adrianmarcu18** [comment]
Ok, so discovered that I was using different zfs versions on the source and destination. Source doesn't have the option of resume, but destination has. I think there should be implemented a way to check if the current zfs version supports resuming, and if not, to ignore the settings from the pyznap config.

**Comment by yboetz** [comment]
Yes, you need a more recent version of ZFS to use resumable send. I might put in a more meaningful error message at some point.

**Comment by adrianmarcu18** [comment]
I think it should be added to the requirements list which is the supported zfs version. Or add an additional check for zfs resume functionality. Because if you end up using resume feature on an older zfs version, there is no way to solve it, than to destroy the destination dataset to be able to send again. So a lot of things can break.
